compiler/circle-mpqsolver/src/CircleMPQSolver.cpp

```diff
@@ -32,6 +32,19 @@ void print_version(void)
   std::cout << vconone::get_copyright() << std::endl;
 }
 
+int handleAutoAlgorithm(arser::Arser &arser, mpqsolver::bisection::BisectionSolver &solver)
```

Comment:
(optional) `handleAutoAlgorithm` -> `checkAutoAlgorithm` ?

Reply:
IMHO the function does not checks but also sets some properties for solver, e.g. sets used `algorithm`.